### PR TITLE
Updated enterprise + analytics link.html + created benchmark page in About

### PR DESCRIPTION
**Pull request by sinzone**

### CODE REVIEW

File: app/enterprise/index.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
           <li class="plan-line muted">
             <h5>commercial plugins</h5>
           </li>
-          <li class="plan-line">Mashape Analytics (Ultra Plan)</li>
+          <li class="plan-line"><a href="https://apianalytics.com" target="_blank">Mashape Analytics</a>(Ultra Plan)</li>
```
Comment (ahmadnassri) [inline app/enterprise/index.html]: use www

Comment (sinzone) [inline app/enterprise/index.html]: we dont have anymore that dns problem.

Comment (ahmadnassri) [inline app/enterprise/index.html]: less redirects ... and stays consistent.

File: app/enterprise/index.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
           <li class="plan-line muted">
             <h5>commercial plugins</h5>
           </li>
-          <li class="plan-line">Mashape Analytics (Ultra Plan)</li>
+          <li class="plan-line"><a href="https://www.apianalytics.com" target="_blank">Mashape Analytics</a>(Ultra Plan)</li>
```
Comment (thibaultCha) [inline app/enterprise/index.html]: It's missing a space before the brace. Also, not analytics.mashape.com?

File: app/docs/0.3.2/about/benchmark.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+title: Benchmark
+---
+
+# Performance Benchmark
+
+We set Kong up on AWS and load tested it to get some performance metrics. The setup consisted of three `m3.medium` EC2 instances; one for Kong, one for Cassandra and a third for an upstream API. After adding the upstream API's `target_url` into Kong we load tested from 1 to 2000 concurrent connections. Complete [reproduction instructions](https://gist.github.com/montanaflynn/01376991f0a3ad07059c) are available and we are currently working towards automating a suite of benchmarks to compare against subsequent releases.
+
+Over two minutes **117,185** requests with an average latency of **10ms** at **976 requests a second** or about **84,373,200 requests a day** went through Kong and back with only a single timeout.
+
+![](http://cl.ly/image/3R171b2U2l3k/Image%202015-06-01%20at%205.00.13%20PM.png)
```
Comment (thibaultCha) [inline app/docs/0.3.2/about/benchmark.md]: We should put the image in the repo. Also I see no link to that page and I don't think it should be under `/docs/` but more under `/about/`

Comment (sinzone) [inline app/docs/0.3.2/about/benchmark.md]: +1

**Comment by sinzone** [comment]
this has been fixed by @thibaultCha

**Pull request closed without merging**